.eslintrc

```diff
@@ -14,13 +14,7 @@
     "import"
   ],
   "rules": {
-    "comma-dangle": ["error", {
```

Comment:
Did you decide to just stick to these rules or are you planning to make more additions/preferences on your eslint. There might be more linting once we start hitting React components

Reply:
I've been going off of these rules, but we should decide on a group linting standard. I'm kind of open to whatever. These were just a few rule changes that I made after watching some videos about the airBnB guide. And, after looking over your PR, I disabled the same func-names rule that you did.